File: backend/Customer/Customer/Models/MessPhoto.cs
using System;
using System.Collections.Generic;

namespace Customer.Models;

public partial class MessPhoto
{
    public int PhotoId { get; set; }

    public int MessId { get; set; }

    public string PhotoUrl { get; set; } = null!;

    public virtual Mess Mess { get; set; } = null!;
}

File: backend/MessOwner/src/main/java/com/example/entities/MessPhoto.java
package com.example.entities;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "mess_photo")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class MessPhoto {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer photoId;

    @JsonIgnore
    @ManyToOne
    @JoinColumn(name = "mess_id", nullable = false)
    private Mess mess;

    @Column(name = "photo_url", nullable = false)
    private String photoUrl;
}
